rllib/offline/offline_prelearner.py

```diff
@@ -96,18 +93,12 @@ def __init__(
         self.config = config
         self.input_read_episodes = self.config.input_read_episodes
         self.input_read_sample_batches = self.config.input_read_sample_batches
-        # We need this learner to run the learner connector pipeline.
-        # If it is a `Learner` instance, the `Learner` is local.
-        if isinstance(learner, Learner):
-            self._learner = learner
-            self.learner_is_remote = False
-            self._module = self._learner._module
-        # Otherwise we have remote `Learner`s.
-        else:
-            self.learner_is_remote = True
-            # Build the module from spec. Note, this will be a MultiRLModule.
-            self._module = module_spec.build()
-            self._module.set_state(module_state)
+        ## Build the module from spec. Note, this will be a MultiRLModule.
```

Comment:
nit nit: remove double `#` and just add typehint: `self._module: MultiRLModule = ...` instead of mentioning the type in the comment.

Reply:
Great catch! Makes totally sense.